packages/kit/test/apps/no-ssr/src/routes/+layout.js

```diff
@@ -1 +1,10 @@
+import { redirect } from '@sveltejs/kit';
+
 export const ssr = false;
+
+export async function load({ route }) {
+	//@ts-ignore
+	if (route.id == '/bad-route' || route.id == '/good-route') {
```

Comment:
`route.id` will always return `null` if the route doesn't exist. We should check for `url.pathname` instead so that this condition passes when we navigate to `/bad-route`.